Classes/Bookmark/BookmarkStore.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  BookmarkStore.swift
+//  Freetime
+//
+//  Created by Rizwan on 18/10/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+final class BookmarksStore {
+    static let shared = BookmarksStore()
```

Comment:
This will share bookmarks between accounts, wont it? Is that how we want it to work? Might be a little weird swapping between accounts and forgetting who you're commenting as b/c the content all looks the same.

I'd also prefer not using singletons as much as physically possible. Dependency injection ftw.